src/include/daos_obj.h

```diff
@@ -900,6 +900,63 @@ daos_obj_anchor_split(daos_handle_t oh, uint32_t *nr, daos_anchor_t *anchors);
 int
 daos_obj_anchor_set(daos_handle_t oh, uint32_t index, daos_anchor_t *anchor);
 
+/**
+ * Open Object Index Table (OIT) of an container
+ *
+ * \param[in]	coh	Container open handle.
+ * \param[in]	epoch	epoch of a snapshot
+ * \param[out]	oh	Returned OIT open handle.
+ * \param[in]	ev	Completion event, it is optional and can be NULL.
+ *			The function will run in blocking mode if \a ev is NULL.
+ *
```

Comment:
BTW, we don't recreate the object ID table if it already exists, right?

Reply:
It is just an object, we could potentially write to it again, but if we store extra information in RDB, then we could skip re-create.